caffe2: Fix lint errors in native/xnnpack/Linear.cpp

EricGriffith opened the pull request:
Summary: See title

Test Plan: Sandcastle

Differential Revision: D72275403

malfet reviewed (approved):
Sure, if OSS lint is green

EricGriffith committed:
caffe2: Fix lint errors in native/xnnpack/Linear.cpp (#150508)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/150508

See title

Test Plan: Sandcastle

Reviewed By: malfet

Differential Revision: D72275403
